lib/fx/adapters/postgres.rb

```diff
@@ -128,7 +144,8 @@ def drop_function(name)
         if support_drop_function_without_args
           execute "DROP FUNCTION #{name};"
         else
-          execute "DROP FUNCTION #{name}();"
+          signature = function_signature_by_name name
+          execute "DROP FUNCTION #{signature};"
```

Comment:
You can get inspiration from this:
https://dba.stackexchange.com/questions/122742/how-to-drop-all-of-my-functions-in-postgresql